tests/test_embedding.py

```diff
@@ -0,0 +1,29 @@
+import finalfusion
+
+
+def test_embeddings():
```

Comment:
I am not sure if I like this test. First, the tests for the normal iterator should be separated from the test with the norms iterator. Second, I don't really like these string-wise comparisons. If Python changes the `__str__` representation of any of the types, the unit tests will break.

I think it is better to create an artificial fifu embedding file with known values. E.g., you could start with a simple text embedding file, such as:

```
foo 1.0 2.0 3.0
bar 4.0 5.0 6.0
baz 7.0 8.0 10.0
```

and convert it to `fifu` using `ff-convert`. When iterating over the embeddings, you can verify the exact values. You can also precompute the norms and verify that they match with pytest's `approx`:

https://docs.pytest.org/en/latest/reference.html#pytest-approx